tasks/ss-websrv-gunicorn.yml

```diff
@@ -78,4 +78,8 @@
   with_items:
     - "archivematica-storage-service"
     - "nginx"
+  environment:
+    LANG: "en_US.UTF-8"
+    LC_ALL: "en_US.UTF-8"
+    LC_LANG: "en_US.UTF-8"
```

Comment:
Is this necessary? I don't think the `service` module is going to make use of your environment.